README.md

```diff
@@ -67,19 +67,24 @@ dut:
   interface:
     pf1:
       name: "ens7f0"        # first PF interface name
+      pci: "0000:00:00.0"   # first PF PCI address
```

Comment:
we used to have the "pci" info in the yaml file, then we decided we could derive this info from the interface name; shall we use that util function to get this pci info?

Reply:
Addressed in the latest commit